### DISCUSSION TITLE
React pdf link

### DISCUSSION THREAD

**Discussion by poshan-kandid** [diegomura/react-pdf#1724] [Q&A]
how to open react pdf's link in new tab?

**Comment by jeetiss** [comment]
hi, @poshan-kandid there are no external links in the pdf specification, so these are options for pdf viewers. Chrome opens the pdf link in the same tab, other pdf viewers open the default browser when you clicked on a link.

**Reply by poshan-kandid** [reply]
In the pdf there is a link ("<"Link">") which I want to open on the new tab like an anchor tag with target="_blank"

**Reply by nalliot** [reply]
It appears it is not possible as it is. The `setLink` method is not setting the props.target in anyway so we cannot set target. 

Please note that while target is not a standard pdf link attribute, it is interpreted by most browser built-in pdf viewers in 2023.
